lib/decred/cw_decred.dart

```diff
@@ -103,5 +103,16 @@ class CWDecred extends Decred {
     return decredWallet.unspents();
   }
 
+  @override
   void updateUnspents(Object wallet) async {}
+
+  @override
+  int heightByDate(DateTime date) {
+    final genesisBlocktime =
+        DateTime.fromMillisecondsSinceEpoch(1454954400 * 1000);
+    final minutesDiff = date.difference(genesisBlocktime).inMinutes;
+    // Decred has five minute blocks on mainnet.
+    // NOTE: This is off by about a day.
+    return (minutesDiff / 5).toInt();
+  }
```

Comment:
Is this not crucial to be accurate? Seems like it would be pretty off considering there was that change in mining algorithm and maybe some other events I don't know about. Would it be possible to first use this as an estimate, then look up the block header, check the date, and get closer from there? Or do you not have the headers at this point?

Reply:
We don't have the headers no.

hmm...

Reply:
Wait I'm wrong. I was thinking this was during restore but it's not. Looking at this again now.

Reply:
Adding a TODO for now. https://github.com/JoeGruffins/cake_wallet/compare/90197856dec93766f62b9e4ca12c3c10e57491fc..1482066fa2e3f697563f421a958f5c9187589bd7

imo we can just remove the choice altogether and use the wallet birthday.